cmd/api/app/api.go

```diff
@@ -141,3 +144,16 @@ func serve(opts *options.Options) {
 		klog.Fatal(router.Router().Run(insecureAddress))
 	}()
 }
+
+
+// Custom function to initialize the application
+func web_terminal() {
+	// Your existing setup code...
+	// Ensure that the terminal setup routes are registered
+	terminal_setup.Init()// Call the init function if it exists
+}
+
+// Call the web_terminal function from the main entry point
```

Comment:
the main file is useless, i seems that it just for testing, testing code expect UT code should not be submitted.